src/meta/test/CMakeLists.txt

```diff
@@ -182,11 +182,11 @@ add_executable(
         $<TARGET_OBJECTS:raftex_obj>
         $<TARGET_OBJECTS:raftex_thrift_obj>
         $<TARGET_OBJECTS:wal_obj>
-	$<TARGET_OBJECTS:thrift_obj>
+	    $<TARGET_OBJECTS:thrift_obj>
```

Comment:
I see it's still not aligned

Reply:
My bad, resolved.  Thanks for your careful review @laura-ding